src/index.js

```diff
@@ -81,6 +83,8 @@ class ReactNativeModal extends Component {
     backdropOpacity: 0.7,
     backdropTransitionInTiming: 300,
     backdropTransitionOutTiming: 300,
+    deviceWidth:  Dimensions.get("window").width,
+    deviceHeight: Dimensions.get("window").height,
```

Comment:
Let's not add any default for them. 
If both the `deviceWidth` and `deviceHeight` prop are supplied to the component we will use them, otherwise we just use the ones from the `state`.

Reply:
In state also we defined the same value for these height & width so I think this will work for both whether we need to use their default value or one supplied through prop.

Reply:
If we supply a default prop though we won't be able to detect if the user has set them or if they're just the defaults.